src/app/visual/card/card-visual.component.html

```diff
@@ -122,4 +122,31 @@
   </sky-card>
 </div>
 
+<div
+  id="screenshot-card-inline-delete"
+>
+  <sky-card
+    [selectable]="'false'"
+  >
+    <sky-card-title>
+      Card title
+    </sky-card-title>
+    <sky-card-content>
+      Card content
+    </sky-card-content>
+    <sky-card-actions>
+      <button
+        class="sky-btn sky-btn-danger"
+        id="inline-delete-trigger"
+        type="button"
+        (click)="triggerInlineDelete()"
+      >
+        Delete
+      </button>
+    </sky-card-actions>
+    <sky-inline-delete *ngIf="showInlineDelete"
```

Comment:
After clicking `Delete` or `Cancel`, the card still appears grey-ed out and bordered red. Is something missing?

Reply:
I just didn't fully wire everything up. Fixed.

Reply:
@Blackbaud-TrevorBurch looks better, but here's another thing - Now when I use keyboard controls to activate the delete button, focus doesn't move. If I hit "Enter" again, its lets me activate the underlying "Delete" button. Should it? FWIW, I can also tab into the checkbox of the card and interact with it. Should this block keyboard controls and place focus on the new Delete/Cancel buttons?

Reply:
Hmmm 👀

Reply:
Fixed